bots/install/test/process.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
This file should be outside the microservice. `${systemDirectory}/config/leo-bus/process.js`
We are trying to keep stack specific stuff out of the public repo

Reply:
I just put in dummy data